src/components/TableStyle.css

```diff
@@ -0,0 +1,182 @@
+div {
+     margin:  auto; 
+    margin-top: 5px;
+    max-width: 1200px;
+    font-family: 'Arial', sans-serif;
+    background-color: #fff;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .heading {
+    text-align: center;
+    color: #333;
+    font-size: 28px;
+    margin-bottom: 20px;
+  }
+  
+  .filter-container {
+    width: 80%;
+    max-width: 800px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    /* margin-bottom: 20px; */
+  }
+  
+  label {
+    font-size: 16px;
+    margin-right: 10px;
+    color: #555;
+  }
+  
+  input[type="date"] {
+    padding: 10px;
+    font-size: 14px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    width: 150px;
+    margin-right: 10px;
+    max-width: 200px;
+  }
+  
+  .btn {
```

Comment:
.btn and .back-btn styles are identical, except for hover colors.
You can merge them with a shared class or use a more specific selector for hover.